west.yml

```diff
@@ -215,7 +215,7 @@ manifest:
       groups:
         - debug
     - name: sof
-      revision: 87de994f49b37e385e748e47b769140df43451cf
+      revision: pull/18/head
```

Comment:
The pull req is merged, but I guess this still works.

Reply:
no, I need now to update with the real commit, so please come back and review again once it is in.